apps/vr-tests/src/stories/Breadcrumb.stories.tsx

```diff
@@ -1,6 +1,6 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 import * as React from 'react';
-import Screener, { Steps } from 'screener-storybook/src/screener';
+import Screener, { Steps, Keys } from 'screener-storybook/src/screener';
```

Comment:
Remove `Keys` import is I don't see it used anywhere. Also, I believe you can remove `Steps` too and leave code line `12` as it was before.

Reply:
Thank you for catching the `Keys`. I was attempting to use it for the focus, but realized it was not working. 

On the note of the `Steps` import, I noticed that most of the `*.stories.tsx` files were formatted in that fashion, which is why I added it to standardized the format of the files. Should I still remove it?

Reply:
Yeah... if you look at [this PR](https://github.com/OfficeDev/office-ui-fabric-react/pull/7757) you will see that it's doing some cleanup in the stories so I believe removing it should be fine... this will create some conflicts though for someone of you depending on which PR gets merged first)))

Reply:
Will do. Thanks!